pizza_cutter/des_pizza_cutter/_pizza_cutter.py

```diff
@@ -212,11 +212,27 @@ def make_des_pizza_slices(
     validate_meds(meds_path + '.fz')
 
 
-def _make_epochs_info(object_data, weights, all_se_info):
+def _make_epochs_info(object_data, weights, all_slices, all_se_info):
+    """
+    record info for each epoch we considered
+
+    object_data: array
+        object is really a coadd
+    weights: array
+        The weights for used images, can be zero length
+    all_slices: list of slices
+        All the considered slices
+    all_se_info: list of dict
+        info for all considered slices
+    """
     dt = [
         ('id', 'i8'),
         ('file_id', 'i8'),  # index into image info
         ('flags', 'i4'),
+        ('row_start', 'i8'),
+        ('col_start', 'i8'),
+        ('box_size', 'i8'),
+        ('psf_box_size', 'i8'),
```

Comment:
The PSF image starts at a different row and col than the slice. We might want to save that info too.